src/com/linkedin/parseq/ParSeqHelper.java

```diff
@@ -0,0 +1,28 @@
+package com.linkedin.parseq;
+
+import com.linkedin.parseq.promise.Promises;
+import com.linkedin.parseq.promise.SettablePromise;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CompletionStage;
+
+public class ParSeqHelper {
+  /**
+   * Helper method to create a Task instance from a ()->CompletionStage
+   */
+  public static <T> Task<T> toTask(final Callable<CompletionStage<? extends T>> callable)
```

Comment:
I know we discussed it and I suggested to create a helper class but in current form it will be hard to discover this new method.
Could you move this method to Task interface and name it "fromFuture"?
Also please provide two forms: one that accepts Task's description and one that generates it automatically e.g.:
```
public static <T> Task<T> fromFuture(final String desc, final Callable<CompletionStage<? extends T>> callable) {
...
}

public static <T> Task<T> fromFuture(final Callable<CompletionStage<? extends T>> callable) {
  return fromFuture("fromFuture: " + _taskDescriptor.getDescription(callable.getClass().getName()), callable);
}
```

Reply:
Got it, this structure makes more sense. I have uploaded the new "fromFuture"